File: C++/BankCartC++/BankAccount.cpp
#include "stdafx.h"
#include <iostream>
#include "IPayment.h"
using namespace std;

class BankAccount : public IPayment
{
private:
	double StateMyAccount = 10000;

public:
	double IPayment::CheckStateAccount()
	{
		return StateMyAccount;
	}

public:
	void IPayment::PayOutMoneyFromAccount(double AmountPayedOutMoney)
	{
		StateMyAccount = StateMyAccount - AmountPayedOutMoney;
	}
};

File: Java/BankJava/BankAccount.java
package bankjava;

class BankAccount implements IPayment {
	
	private double StateMyAccount= 10000;
	 
    public double CheckStateAccount()
    {
        return StateMyAccount;
    }
 
    public void PayOutMoneyFromAccount(double AmountPayedOutMoney)
    {
        StateMyAccount = StateMyAccount - AmountPayedOutMoney;
    }
}
